src/serac/physics/solid.hpp

```diff
@@ -200,6 +200,9 @@ class Solid : public BasePhysics {
    * @param[in] keep_deformation Flag to keep the deformation in the underlying mesh post-destruction
    * @param[in] name An optional name for the physics module instance
    * @param[in] pmesh The mesh to conduct the simulation on, if different than the default mesh
+   *
+   * @note @p name is the physics module name and not the mesh tag to be used. If a non-default mesh is desired, the
+   * mesh pointer must be provided. Otherwise, it attempts to find the "default" mesh in the data store.
    */
   Solid(int order, const SolverOptions& options, GeometricNonlinearities geom_nonlin = GeometricNonlinearities::On,
         FinalMeshOption keep_deformation = FinalMeshOption::Deformed, const std::string& name = "",
```

Comment:
re: the confusion, it might also be worth changing the `name` argument to `instance_name` or similar.

Reply:
Semantically, is there a distinction between the terms "name" and "tag" (e.g., `mesh_tag`) used in the API?